app/mailers/application_mailer.rb

```diff
@@ -1,4 +1,4 @@
 class ApplicationMailer < ActionMailer::Base
-  default from: "from@example.com"
-  layout "mailer"
+  default from: 'from@example.com'
+  layout 'mailer'
```

Comment:
### Good work creating a new repository for the back end. 

*There are some things that you need to do in the way to connect both applications, front-end, and back-end.*

**Kindly as in the requirements is requested. Follow the steps to complete the back-end connection**

- Make sure that your project has a Postgres database set up.
- Create a table for storing your messages and create 5 different greetings.
- Create an API endpoint that selects a random greeting from your table (you will need a controller with an action and rails route).

Reply:
Everything you mentioned is implemented. It has a postgres database, has a table and the endpoint is present. Kindly clone and run appropriately. Thank you